packages/trpc/server/routers/loggedInViewer/updateProfile.handler.ts

```diff
@@ -31,12 +32,36 @@ type UpdateProfileOptions = {
   input: TUpdateProfileInputSchema;
 };
 
+const uploadAvatar = async ({ userId, avatar: data }: { userId: number; avatar: string }) => {
+  const objectKey = uuidv4();
+
+  await prisma.avatar.upsert({
+    where: {
+      teamId_userId: {
```

Comment:
Ideally we'd be able to use `null` for teamId in this situation but this is something Prisma does not support. Union types are non-nullable. In this situation that is OK as we can use `0` instead.